deps/libcaption/src/caption.c

```diff
@@ -408,10 +408,11 @@ size_t caption_frame_dump_buffer(caption_frame_t* frame, utf8_char_t* buf)
 {
```

Comment:
`snprintf` can't be safely used in this function without the size of the buffer being passed as a function.

Reply:
The buffer is initialised with `CAPTION_FRAME_DUMP_BUF_SIZE`, so it should suffice to carry on with that value?

Reply:
That should be OK, I would probably also mark the function as `static` to ensure only callers in caption.c (that presumably are aware of the buffer size) can use it.

Reply:
Removed both functions as discussed.

Reply:
Has this been resolved? Would like to move this forward.

Reply:
Yeah should be resolved as the corresponding functions have been removed entirely.